skbio/io/format/genbank.py

```diff
@@ -435,20 +422,38 @@ def _serialize_single_genbank(obj, fh):
     Always write it in NCBI canonical way:
     1. sequence in lowercase
     2. 'u' as 't' even in RNA molecules.
+
+    Parameters
+    ----------
+    obj : Sequence or its child class
+
     '''
+    # write out the headers
     md = obj.metadata
     for header in _HEADERS:
         if header in md:
             serializer = _SERIALIZER_TABLE.get(
                 header, _serialize_section_default)
-            out = serializer(header, md[header])
-            # test if 'out' is a iterator.
-            # cf. Effective Python Item 17
-            if iter(out) is iter(out):
-                for s in out:
+            if header == 'FEATURES':
+                indent = 21
```

Comment:
Maybe add a comment explaining where the number came from? i.e. what you just explained to me about it anecdotally being used in genbank files, and that the number isn't strict (can be 20 or 22, maybe other values?). I could see a future maintainer wondering about why 21 is used as an indent.